README_SETUP.md

```diff
@@ -214,6 +224,11 @@ Download and install using `apt-get`:
 ```sh
 > sudo apt-get install -y --no-install-recommends libssl-dev openssl libtool autoconf automake build-essential uuid-dev libxi-dev libopenal-dev libgl1-mesa-dev libglw1-mesa-dev freeglut3-dev libncurses5
 ```
+
+Download and install using `dnf`:
+```
+> sudo dnf install python3-pip python3-cffi python3-setuptools openssl-devel openssl libtool autoconf automake gcc gcc-c++ clang glibc-devel make libuuid-devel libXi-devel openal-soft-devel mesa-libGL-devel mesa-libGLw-devel freeglut-devel ncurses-devel ncurses-compat-libs
```

Comment:
I don't like one package manager installing more packages than the other?
We should strive for making sure the default package manager (apt) should work.